src/whisper_at_train/run_combined_training.sh

```
@@ -1,85 +1,154 @@
-source /home/taemyung_heo/.cache/pypoetry/virtualenvs/whisper-at-z6hdRBdT-py3.10/bin/activate
-export TORCH_HOME=../../pretrained_models
+# 가상환경 활성화
+# Poetry 환경이 있다면 자동으로 찾아서 활성화
+if command -v poetry &> /dev/null && [ -f "pyproject.toml" ]; then
+    echo "Poetry 환경 활성화 중..."
+    poetry shell
+else
+    # 수동 가상환경 경로 (환경에 맞게 수정 필요)
+    VENV_PATH="${HOME}/.cache/pypoetry/virtualenvs/whisper-at-z6hdRBdT-py3.10/bin/activate"
+    if [ -f "$VENV_PATH" ]; then
+        source "$VENV_PATH"
```

Shellcheck (0.10.0)

[warning] ShellCheck can't follow non-constant source. Use a directive to specify location.

(SC1090)